SU-HSDO/suhumsci
# SHS-5996/SHS-5997: WYSIWYG link styles with arrow or external-link show green icon instead of matching the colour palette / WYSIWYG link style "External Link" does not show external icon

# READY FOR REVIEW

## Summary
- WYSIWYG link styles with arrow or external-link icon color matches the colour palette
- WYSIWYG link style "External Link" shows external icon

## Need Review By (Date)
12/20

## Urgency
medium

## Steps to Test
1. Go to [Colorful](https://hs-colorful-wjssscvgy8xojuxe8r2anjaxhxn5homo.tugboatqa.com/test-links) and [Traditional](https://hs-traditional-wjssscvgy8xojuxe8r2anjaxhxn5homo.tugboatqa.com/test-links)
2. Or create a flexible page and add a WYSIWYG block with a "More link" style and a "External link" sty;e
3. Confirm the icon color matches the default colour palette (cardinal) for the "More link"
4. Confirm the style matches as expected in the "External link" style preview

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209090732666815
